hybrasyl/Messaging/Group.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Hybrasyl.Objects;
+
+
+namespace Hybrasyl.Messaging
+{
+
+    class GroupCommand : ChatCommand
+    {
+        public new static string Command = "group";
+        public new static string ArgumentText = "<string username>";
+        public new static string HelpText = "Invite the specified player to your group.";
+        public new static bool Privileged = false;
+
+        public new ChatCommandResult Run(User user, params string[] args)
+        {
+            User newMember = Game.World.FindUser(args[0]);
```

Comment:
If the user doesn't exist, I'm getting KeyNotFoundException in WorldDataStore (line 63)
![image](https://user-images.githubusercontent.com/33741983/35200333-0fa1da02-fec1-11e7-8462-82b92f05d905.png)